lochness/xnat/__init__.py

```diff
@@ -10,36 +10,41 @@
 import lochness.tree as tree
 import lochness.config as config
 
-yaml.SafeDumper.add_representer(col.OrderedDict, yaml.representer.SafeRepresenter.represent_dict)
+yaml.SafeDumper.add_representer(
+        col.OrderedDict, yaml.representer.SafeRepresenter.represent_dict)
 
 logger = logging.getLogger(__name__)
 
 @net.retry(max_attempts=5)
 def sync(Lochness, subject, dry=False):
     logger.debug('exploring {0}/{1}'.format(subject.study, subject.id))
 
-    for alias,xnat_uids in iter(subject.xnat.items()):
+    for alias, xnat_uids in iter(subject.xnat.items()):
         Keyring = Lochness['keyring'][alias]
         auth = yaxil.XnatAuth(url=Keyring['URL'], username=Keyring['USERNAME'],
                               password=Keyring['PASSWORD'])
-        for xnat_uid in xnat_uids:
+
+        # CAPS agnostic XNAT data pull
+        for xnat_uid in xnat_uids + [(x[0], x[1].lower()) for x in xnat_uids]:
             for experiment in experiments(auth, xnat_uid):
                 logger.info(experiment)
                 dirname = tree.get('mri',
                                    subject.protected_folder,
                                    processed=False,
                                    BIDS=Lochness['BIDS'])
-                dst = os.path.join(dirname, experiment.label)
+                dst = os.path.join(dirname, experiment.label.upper())
```

Comment:
Saving the output data in the capitalized AMP-SCZ ID output directory, regardless of the XNAT subject ID.